Gemfile

```diff
@@ -144,6 +144,7 @@ group :development, :test do
   gem "amazing_print", "~> 1.2" # Great Ruby debugging companion: pretty print Ruby objects to visualize their structure
   gem "bullet", "~> 6.1" # help to kill N+1 queries and unused eager loading
   gem "capybara", "~> 3.34" # Capybara is an integration testing tool for rack based web applications
+  gem "cypress-rails", "~> 0.4.2" # For end to end tests (E2E)
```

Comment:
New gem to allow us to rollback any changes that occur in a test, i.e. fast data resets.